src/main/java/technology/dice/dicefairlink/discovery/members/awsapi/AwsApiReplicasFinder.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2018 - present by Dice Technology Ltd.
+ *
+ * Please see distribution for license.
+ */
+package technology.dice.dicefairlink.discovery.members.awsapi;
+
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.awssdk.services.rds.model.DBCluster;
+import software.amazon.awssdk.services.rds.model.DBClusterMember;
+import software.amazon.awssdk.services.rds.model.DescribeDbClustersRequest;
+import software.amazon.awssdk.services.rds.model.DescribeDbClustersResponse;
+import software.amazon.awssdk.services.rds.model.DescribeDbInstancesRequest;
+import software.amazon.awssdk.services.rds.model.Filter;
+import software.amazon.awssdk.services.rds.paginators.DescribeDBInstancesIterable;
+import technology.dice.dicefairlink.config.FairlinkConfiguration;
+import technology.dice.dicefairlink.discovery.members.ClusterInfo;
+import technology.dice.dicefairlink.discovery.members.MemberFinderMethod;
+import technology.dice.dicefairlink.driver.FairlinkConnectionString;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class AwsApiReplicasFinder implements MemberFinderMethod {
+  private static final Logger LOGGER = Logger.getLogger(AwsApiReplicasFinder.class.getName());
+  private static final String ACTIVE_STATUS = "available";
+  private static final Set<String> EMPTY_SET = new HashSet<>(0);
```

Comment:
`ImmutableSet.of()`

Reply:
This project doesn't have a dependency on Guava (only on test scope)